embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/spans/SpansServiceImpl.kt

```diff
@@ -161,7 +161,9 @@ internal class SpansServiceImpl(
     override fun flushSpans(appTerminationCause: EmbraceAttributes.AppTerminationCause?): List<EmbraceSpanData> {
         synchronized(completedSpans) {
 
-            val telemetryAttributes = embraceTelemetryService.getTelemetryAttributes()
+            /* Right now, session spans don't survive native crashes and sudden process terminations,
```

Comment:
We should use `//` to mark comments inside the body of a method?

Reply:
But it's a multi-line comment! Wouldn't it be weird to have it like this? 

```
// Right now, session spans don't survive native crashes and sudden process terminations,
// so telemetry will not be recorded in those cases, for now.
```